http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/PersistentDiskComponent.scala

```diff
@@ -70,7 +71,8 @@ class PersistentDiskTable(tag: Tag) extends Table[PersistentDiskRecord](tag, "PE
      diskType,
      blockSize,
      formattedBy,
-     (galaxyPvcId, cvmfsPvcId, lastUsedBy)) <> ({
+     (galaxyPvcId, cvmfsPvcId),
+     lastUsedBy) <> ({
```

Comment:
Clarification question, what was the reason for breaking this out? Was it to allow easier look up in the DB for lastUsedBy?

Reply:
I extracted this out since Cromwell app only needs the `lastUsedBy` and not `galaxyPvcId, cvmfsPvcId` to reconnect to previously used disk when recreating the app.

Reply:
If I understand correctly, the Galaxy "recreate" scenario is actually pause/resume. We are not actually "recreating" Cromwell, we are just creating a new instance using a disk that already exists from a previously deleted instance.

Reply:
Yes that is correct. For Cromwell, `lastUsedBy` is being used to make sure that the Cromwell app that last used the disk and the new Cromwell app that is requesting to reconnect to the disk are in same workspace.
Links:

- [get last used App](https://github.com/DataBiosphere/leonardo/blob/5636dffa3ac58e8a1b1f69ecdcf18fc4a981ceb8/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppComponent.scala#L230-L247)
- [check that workspace is same](https://github.com/DataBiosphere/leonardo/blob/cdd26de1b56a92fa3005f28835b1a2d0edfa4596/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala#L170-L178)